resource.address/http/src/main/java/org/kaazing/gateway/resource/address/http/HttpResourceAddress.java

```diff
@@ -426,6 +440,18 @@ private HttpServerHeaderOption() {
         }
     }
 
+    private static final class HttpUserAgentHeaderOption extends HttpResourceOption<Boolean> {
+        private HttpUserAgentHeaderOption() {
+            super(Kind.USER_AGENT_HEADER, "userAgentHeaderEnabled", Boolean.TRUE);
+        }
+    }
+
+    private static final class HttpHostHeaderOption extends HttpResourceOption<Boolean> {
+        private HttpHostHeaderOption() {
+            super(Kind.HOST_HEADER, "hostHeaderEnabled", Boolean.TRUE);
```

Comment:
It seems like calling the constructor is necessary as there is no valid constructor with no arguments.

Should I remove all instances of HttpHostHeaderOption instead?